spec/domain/audits/save_audit_spec.rb

```diff
@@ -0,0 +1,84 @@
+module Audits
+  RSpec.describe SaveAudit do
+    let!(:user) { create :user }
+    let!(:content_item) { create :content_item, title: "A title" }
+    let!(:filter) { Filter.new }
+    let!(:attributes) {
```

Comment:
You could use `attributes_for` of `FactoryGirl`.

Reply:
Nice! Fixed in https://github.com/alphagov/content-performance-manager/pull/346/commits/55674010858f913a1056174131f50ca1a643551d#diff-dfb8ea57e9196a3e0adca3b88cec560dR5